qtconsole/rich_jupyter_widget.py

```diff
@@ -140,9 +140,8 @@ def _handle_execute_result(self, msg):
                 self._pre_image_append(msg, prompt_number)
                 try:
                     self._append_latex(data['text/latex'], True)
-                except Exception:
-                    self.log.error("Failed to render latex: '%s'", data['text/latex'], exc_info=True)
```

Comment:
yeah, i think we should keep catching all exceptions and only in case of a ValueError (or probably a custom subclass we define) we swallow the error (as this one means that we got non-equation LaTeX)